packages/go_router_builder/example/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   provider: 6.0.5
 
 dev_dependencies:
-  build_runner: ^2.0.0
```

Comment:
Is there a reason this need to be a 2.3.0?

Reply:
If I change it to `build_runner: 2.0.0` and run `flutter clean && flutter pub get`, I get this error:

```
Because every version of go_router_builder from path depends on analyzer >=4.4.0 <6.0.0 and build_runner >=2.0.0 <2.0.6 depends on analyzer ^1.4.0, go_router_builder from path is incompatible
  with build_runner >=2.0.0 <2.0.6.
```

Similar issue with `2.1.0`. However `2.2.0` works fine. Shall I change to `^2.2.0`?

Reply:
I see, this is fine then